local-cli/server/util/launchEditor.js

```diff
@@ -37,6 +37,7 @@ var COMMON_EDITORS = {
   '/Applications/Sublime Text 2.app/Contents/MacOS/Sublime Text 2':
     '/Applications/Sublime Text 2.app/Contents/SharedSupport/bin/subl',
   '/Applications/Visual Studio Code.app/Contents/MacOS/Electron': 'code',
+  '/Applications/WebStorm.app/Contents/MacOS/webstorm': 'webstorm',
```

Comment:
Cool; I didn't realize that's how it's wired up in React-Native. Just out of curiosity, do we need to add support for other IntelliJ products, like RubyMine + IntelliJ? 🤔 

```
/Applications/RubyMine.app/Contents/MacOS/rubymine
```

Reply:
Of course, if people use RubyMine for RN development and it has a command line tool for opening files. I know that webstorm and idea have, but appcode doesn't seem to have one...